Caboodle/Clipboard/Clipboard.uwp.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System;
+using System.Threading.Tasks;
+using Windows.ApplicationModel.DataTransfer;
+
+namespace Microsoft.Caboodle
+{
+    public static partial class Clipboard
+    {
+        public static void SetText(string text)
+        {
+            var dataPackage = new DataPackage();
+            dataPackage.SetText(text);
+            Windows.ApplicationModel.DataTransfer.Clipboard.SetContent(dataPackage);
+        }
+
+        public static bool HasText
+        {
+            get
+            {
+                var clipboardContent = Windows.ApplicationModel.DataTransfer.Clipboard.GetContent();
```

Comment:
Could make expression bodied member here as well, no need to create the var

Reply:
I would do:  
```
public static bool HasText =>
    Windows.ApplicationModel.DataTransfer.Clipboard.GetContent()?.Contains(StandardDataFormats.Text) ?? false;

Reply:
That is going to be a long line, we have a style rule to make sure that it doesn't exceed 120 characters. Maybe make `Windows.ApplicationModel.DataTransfer` a using statement.